docker/stringsifter/build.conf

```diff
@@ -0,0 +1 @@
+version=1.0.0
```

Comment:
would reccomend using dynamic versioning to lock onto stringsifter's version. See how its done in demisto-sdk image